sfont.cpp

```diff
@@ -60,1609 +63,1427 @@ static const bool writeCompressed = true;
 //---------------------------------------------------------
 
 Sample::Sample()
-      {
-      name = 0;
-      }
+{
+    name = 0;
+}
 
 Sample::~Sample()
-      {
-      free(name);
-      }
+{
+    free(name);
+}
 
 //---------------------------------------------------------
 //   Instrument
 //---------------------------------------------------------
 
 Instrument::Instrument()
-      {
-      name = 0;
-      }
+{
+    name = 0;
+}
 
 Instrument::~Instrument()
-      {
-      free(name);
-      }
+{
+    free(name);
+}
 
 //---------------------------------------------------------
 //   SoundFont
 //---------------------------------------------------------
 
 SoundFont::SoundFont(const QString& s)
-      {
-      path      = s;
-      engine    = 0;
-      name      = 0;
-      date      = 0;
-      comment   = 0;
-      tools     = 0;
-      creator   = 0;
-      product   = 0;
-      copyright = 0;
-      }
```

Comment:
You changed the indentation of everything.

The new indentation is also not the MuseScore indentation style (which is a bit weird, but at least consistently used in MuseScore-related code).

Reply:
I'm sorry for that I began to edit the files inside my project and I am used to automatically indent files. You could still reopen the files with an IDE or some advanced text editor for reindenting them.

Reply:
This makes the diff illegible and therefore impossible to review, though.

Reply:
I will not blame you if this pull request is closed without being integrated ;-)
The code is working in Polyphone, that's what I wanted